ci-operator/step-registry/ipi/conf/azuremag/ipi-conf-azuremag-commands.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+REGION="usgovvirginia"
+echo "Azure region: ${REGION}"
+
+workers=3
+if [[ "${SIZE_VARIANT}" == "compact" ]]; then
+  workers=0
+fi
+master_type=Standard_E4s_v3
+if [[ "${SIZE_VARIANT}" == "xlarge" ]]; then
+  master_type=Standard_D32s_v3
+elif [[ "${SIZE_VARIANT}" == "large" ]]; then
+  master_type=Standard_D16s_v3
+elif [[ "${SIZE_VARIANT}" == "compact" ]]; then
+  master_type=Standard_D8s_v3
+fi
+
+DIR=/var/run/azuremag-cluster-secrets
```

Comment:
After my update for osServicePrincipal.json key in vault, I think we do NOT need to write a new osServicePrincipal.json file, we can drop it now.

Reply:
I see the `osServicePrincipal.json` is in vault now and it's in `ci` namespace, so the location is `${CLUSTER_PROFILE_DIR}/osServicePrincipal.json`

Reply:
Yeah, I add it into vault some minutes ago. The default AZURE_AUTH_LOCATION var is already pointed to `${CLUSTER_PROFILE_DIR}/osServicePrincipal.json`. So we can drop the following lines now.